sanger-warren.gemspec

```diff
@@ -27,12 +27,13 @@ Gem::Specification.new do |spec|
   spec.files = Dir.chdir(File.expand_path(__dir__)) do
     `git ls-files -z`.split("\x0").reject { |f| f.match(%r{^(test|spec|features)/}) }
   end
-  spec.bindir        = 'exe'
-  spec.executables   = spec.files.grep(%r{^exe/}) { |f| File.basename(f) }
+  spec.bindir = 'bin'
+  spec.executables << 'warren'
   spec.require_paths = ['lib']
 
   # Dependencies
   spec.add_runtime_dependency 'bunny', '~> 2.17.0'
   spec.add_runtime_dependency 'connection_pool', '~> 2.2.0'
   spec.add_runtime_dependency 'multi_json', '~> 1.0'
+  spec.add_runtime_dependency 'thor', '~> 1.1.0'
```

Comment:
Just noticed, are we missing the reference to the license?

Reply:
Huh, you're right. That's odd, I used the gem templating built into bundler, which itself generated the license. I wonder why it doesn't reference it here.